module/documents/token.mjs

```diff
@@ -18,6 +20,7 @@ export default class TokenDocument5e extends TokenDocument {
 
   /** @inheritdoc */
   static getTrackedAttributes(data, _path=[]) {
+    if ( data instanceof foundry.abstract.DataModel ) return this._getTrackedAttributesFromSchema(data.schema, _path);
```

Comment:
Looks good, but I wonder if it would be better for this logic to live as a static method on our data models themselves, so you would instead do:
```js
if ( data instanceof SystemDataModel ) data.constructor.getTrackedAttributes()
```

Reply:
I think that would work fine. Let me submit my PR for the core half of this work, as it may impact what we decide to do here. But it would still work if we take over the process in `TokenDocument5e.getTrackedAttributes` and redirect it to `SystemDataModel`.